src/decision/include/GpsDecision.h

```diff
@@ -14,15 +14,51 @@
 #include <geometry_msgs/Twist.h>
 #include <geometry_msgs/Point.h>
 #include <ros/ros.h>
+#include <std_msgs/Float32.h>
 
 class GpsDecision {
 public:
     GpsDecision(int argc, char **argv, std::string node_name);
-private:
+
+    /**caclutes the distance between wayPoint and current point
+     * @ parameter: given the gps locaiton of the next move
+     * @ return: the distance between the next location and the current location
+     **/
+    static double distance(const geometry_msgs::Point::ConstPtr& relative_gps);
+
+    /**
+     * @ parameter
+     * relative_gps: given the gps locaiton of the next move
+     * current_heading: the curren heading relative to north in degrees (0 to 360 degrees)
+     * @ return:
+     *   desiredAngle: the angle(less than PI) in degrees that robot need to turn
+     *   positive value means turning clockwise
+     *   negative value means turing counter-clockwise
```

Comment:
I believe that the variable and initial description should be on the same line. Also, you don't need a variable name for the return value unless it actually adds meaning to the documentation. So this would be more like
```
@return the angle(less than PI) in degrees that robot need to turn
positive value means turning clockwise
negative value means turing counter-clockwise
```